### PR TITLE
Create detailed board screen

### PR DESCRIPTION
**Pull request by ChiuMungZitAlexander**

### CODE REVIEW

File: src/client/components/boardDetail/index.js [hunk 1]
```diff
@@ -40,11 +67,14 @@ class BoardDetail extends React.Component {
   generateBoardColumn() {
     let boardDisplay = [];
 
-    this.state.data.columns.forEach(function(element) {
-      boardDisplay.push(
-        <BoardColumn column={element} key={element._id} />
-      );
-    }, this);
+    if (Object.prototype.toString.call(this.state.data.columns)
```
Comment (beltekylevi) [inline src/client/components/boardDetail/index.js]: can it be different from an Array? if you want to check if something is an array: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/isArray

**Review by beltekylevi — CHANGES_REQUESTED** [review]
also which story does it solve?

File: src/client/components/boardDetail/index.js (hunk 1 above)
Comment (ChiuMungZitAlexander) [reply src/client/components/boardDetail/index.js]: good idea

**Comment by ChiuMungZitAlexander** [comment]
to solve BLP-35 Create detailed board screen

File: src/client/components/boardDetail/index.js (hunk 1 above)
Comment (beltekylevi) [reply src/client/components/boardDetail/index.js]: so please if `this.state.data.columns`'s type can be different from array please check it with the built in `isArray` method! :)

**Review by beltekylevi — CHANGES_REQUESTED** [review]

**Review by beltekylevi — APPROVED** [review]

**Comment by ChiuMungZitAlexander** [comment]
luckily both are alright, while Object.prototype.toString.call(arg) === '[object Array]' is compatible to ECMA5 or older version. We should keep the pace with ES6 ES7 🤓🤓🤓

**Comment by beltekylevi** [comment]
i'm glad that you did your research! 🙂

**Pull request merged**